# oteodoro changes:
# Changed Options.td for documentation
# For use for the distro's hardened profile
diff -urp a/include/clang/Driver/Options.td b/include/clang/Driver/Options.td
--- a/include/clang/Driver/Options.td	2021-06-28 09:23:38.000000000 -0700
+++ b/include/clang/Driver/Options.td	2021-09-28 18:19:17.986970417 -0700
@@ -5116,7 +5116,7 @@ def stack_protector : Separate<["-"], "s
   HelpText<"Enable stack protectors">, Values<"0,1,2,3">,
   NormalizedValuesScope<"LangOptions">,
   NormalizedValues<["SSPOff", "SSPOn", "SSPStrong", "SSPReq"]>,
-  MarshallingInfoString<LangOpts<"StackProtector">, "SSPOff">, AutoNormalizeEnum;
+  MarshallingInfoString<LangOpts<"StackProtector">, "SSPStrong">, AutoNormalizeEnum;		// Changed by oteodoro for documentation consistency
 def stack_protector_buffer_size : Separate<["-"], "stack-protector-buffer-size">,
   HelpText<"Lower bound for a buffer to be considered for stack protection">,
   MarshallingInfoStringInt<CodeGenOpts<"SSPBufferSize">, "8">;
